.evergreen/ssdlc-report.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -o errexit
+
+############################################
+#            Main Program                  #
+############################################
+RELATIVE_DIR_PATH="$(dirname "${BASH_SOURCE[0]:-$0}")"
+source "${RELATIVE_DIR_PATH}/javaConfig.bash"
+
+echo "Creating SSLDC reports"
+./gradlew -version
+./gradlew -PssdlcReport.enabled=true --continue -x test -x integrationTest -x spotlessApply clean check scalaCheck kotlinCheck testClasses || true
```

Comment:
This command most likely fails due to SpotBugs findings, because it ignores `spotbugs/exclude.xml`. To prevent the whole script from failing, we use `|| true`.